i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-standard-snmp.md

```diff
@@ -137,91 +302,866 @@ Dans l'exemple ci-dessus, la communauté choisie est 'public'. Cette valeur est
+| FILTERPORT     | Filter on port description (can be a regexp)                                                                                                                                                                                         | .*                             |             |
+| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (Default: '%{op\_status} =~ /up/ && %{state} =~ /blocking\|broken/'). You can use the following variables: %{state}, %{op\_status}, %{admin\_status}, %{port}, %{index} | %{state} =~ /blocking\|broken/ |             |
+| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING. You can use the following variables: %{state}, %{op\_status}, %{admin\_status}, %{port}, %{index}                                                                       |                                |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                                                                                                  | --verbose                      |             |
 
-Lorsque vous ajoutez un hôte, complétez les champs 'Communauté Snmp' et 'Version Snmp' pour refléter la configuration de l'équipement cible. 
+</TabItem>
+<TabItem value="Stack" label="Stack">
+
+| Macro                   | Description                                                                                                                                                                                                                                                                                                                                                                                        | Valeur par défaut                                | Obligatoire |
+|:------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------------------|:-----------:|
+| WARNINGADDED            | Set thresholds on members count for each states. (                                                                                                                                                                                                                                                                                                                                                 |                                                  |             |
```

Comment:
The full help message was 
```
    --warning-* --critical-*
            Set thresholds on members count for each states. (Can be:
            'waiting', 'progressing', 'added', 'ready', 'sdm-mismatch',
            'version-mismatch', 'feature-mismatch', 'new-master-init',
            'provisioned', 'invalid', 'removed')
```
So it will just be "Thresholds"